examples/templatebuilder/etcd-template.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
```

Comment:
nit: make this file as simple as it can be so that the intent of the example is clearer.

Reply:
Probably followup to do this since all the examples use `etcd`

Reply:
https://github.com/GoogleCloudPlatform/k8s-cluster-bundle/issues/307